src/cart/src/include/gurt/types.h

```diff
@@ -97,6 +97,20 @@ typedef struct {
 	d_iov_t		*sg_iovs;
 } d_sg_list_t;
 
+/**
+ * c string buffer
+ */
+typedef struct {
```

Comment:
(style)  do not add new typedefs

Reply:
I am wondering if we should keep this. This structure should be manipulated only using the new functions added on the current patch

Reply:
In general, we don't add new ones but I'm indifferent